### PR TITLE
Add setBuffer() method to OutputMessage class

### PR DESCRIPTION
**Pull request by EgzoT**
This method similar to method setBuffer of InputMessage class will able to easy data copy from one OutputMessage/InputMessage to other.

May be helpful for make cast system using TCP/IP protocol.

### CODE REVIEW

File: src/framework/net/outputmessage.h [hunk 1]
```diff
@@ -40,6 +40,7 @@ class OutputMessage : public LuaObject
 
     void reset();
 
+	void setBuffer(const std::string& buffer);
```
Comment (ubap) [inline src/framework/net/outputmessage.h]: incorrect identation.
tab -> spaces

File: src/framework/net/outputmessage.cpp [hunk 2]
```diff
@@ -35,6 +35,16 @@ void OutputMessage::reset()
     m_messageSize = 0;
 }
 
+void OutputMessage::setBuffer(const std::string& buffer)
+{
+	int len = buffer.size();
```
Comment (ubap) [inline src/framework/net/outputmessage.cpp]: incorrect identation.
tabs -> spaces

**Comment by EgzoT** [comment]
@ubap Thx, for catch this.
In fact I always forget about indentation, I must remember about indentations in future.

**Comment by hrsantiago** [comment]
@EgzoT however you PR now is a mess :disappointed: would you mind sending a new one with only one commit?

**Comment by EgzoT** [comment]
@hrsantiago I added New push request, with one commit.
#967

**Pull request closed without merging**